### PR TITLE
container.py fix to add get_detail

### PR DESCRIPTION
**Pull request by pavelzag**

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 914e6dbfd14a2709ea32df5d0cf116f32dd9f6fb:

Everything seems all right :smile: .
*CFME QE Bot*

File: cfme/containers/container.py [hunk 1]
```diff
@@ -49,3 +49,13 @@ def navigate(self, detail=True):
     def click_element(self, *ident):
         self.load_details(refresh=True)
         return sel.click(details_page.infoblock.element(*ident))
+
+
+def get_detail(self, *ident):
```
Comment (psav) [inline cfme/containers/container.py]: Needs indenting to become part of the class

**Comment by dajoRH** [comment]
Lint report for commit 09c904d1d8293238faf34fe166835393f30c087f:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by psav** [comment]
Squash and this one can be merged

**Comment by dajoRH** [comment]
Lint report for commit f64479527dcc231087d6ba1525b8e1adde427878:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 76d0632cb1261958e1a095e080becaa1789d7c9a:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit be18af933c0d39ff4bd1538195fa773e444e3bd4:

Everything seems all right :smile: .
*CFME QE Bot*

**Pull request merged**